src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -376,6 +378,28 @@ impl Annotation {
         self.get(CONTAINER_TYPE_KEY)
     }
 
+    /// Get the annotation of cpu quota for sandbox
+    pub fn get_sandbox_cpu_quota(&self) -> u64 {
+        let value = self
```

Comment:
Nit:
Chnage

```rust
        let value = self
            .get_value::<u64>(SANDBOX_CPU_QUOTA_KEY)
            .unwrap_or(Some(0));
        value.unwrap_or(0)
```
to 
```rust
       self
            .get_value::<u64>(SANDBOX_CPU_QUOTA_KEY)
            .unwrap_or(Some(0)).unwrap()
```
to avoid using an intermedia variable.

Reply:
Thanks @liubin , nice catch. Fixed.